AdaptiveUiCodelab/app/src/main/java/com/example/reply/ui/ReplyApp.kt

```diff
@@ -67,7 +88,8 @@ private fun ReplyNavigationWrapperUI(
                     onClick = { /*TODO update selection*/ },
                 )
```

Comment:
Should we actually update the selected item? It actually introduces a bug by reverting back to Inbox after a config change, and the codelab can point this out and implement a fix (using `rememberSaveable` on L66).

Reply:
Would we show anything different when the destination changes? It might be weird if the button works, but doesn't change anything.

And if we did want to show something different, we'd probably want to use the navigation component to do it properly.

Reply:
> Would we show anything different when the destination changes? It might be weird if the button works, but doesn't change anything.
> 
> And if we did want to show something different, we'd probably want to use the navigation component to do it properly.

I wasn't intending to show anything different, just point out the importance of recognizing which state needs to be retained and making sure to do that. Agree that it's a bit weird, but we can give an explanation in the codelab text (e.g. "the app currently does not change its content based on this selection, but you could implement that by reading this value and calling different Composables, or mapping the values to routes in a NavHost").

Reply:
Got it, with that caveat as a comment I think pointing out where we should be using `rememberSaveable` sounds good to me.